cost-analyzer/templates/_helpers.tpl

```diff
@@ -902,7 +902,7 @@ Begin Kubecost 2.0 templates
     periodSeconds: {{ .Values.kubecostAggregator.readinessProbe.periodSeconds }}
     failureThreshold: {{ .Values.kubecostAggregator.readinessProbe.failureThreshold }}
   {{- end }}
-  imagePullPolicy: Always
+  imagePullPolicy: IfNotPresent
```

Comment:
Should we use this as an opportunity to template the `imagePullPolicy` here on the Aggregator and also the Forecasting image? Overall this looks good! Just want to make sure that if we change the current default behavior the user has the option to override.

Reply:
I knew you were going to ask that :) I agree, but I'd like some help with that from someone else if possible.

Reply:
Haha sure thing! I'll try to commit those changes onto this PR later today.

Reply:
There's money to be made in an AI specific helm templater :)